app/models/course/assessment/submission_question.rb

```diff
@@ -23,6 +23,18 @@ def notify(post)
     Course::Assessment::SubmissionQuestion::CommentNotifier.post_replied(post.creator, post)
   end
 
+  def answers
+    Course::Assessment::Answer.where('submission_id = ? AND question_id = ?',
+                                     self.submission_id, self.question_id)
+  end
+
+  # Loads the past answers of a specific question
+  def past_answers(num_answers_to_load)
```

Comment:
Instead of setting default below, can use [default params](https://robots.thoughtbot.com/ruby-2-keyword-arguments.)